setup.py

```diff
@@ -43,8 +43,8 @@ def finalize_options(self):
             sys.exit(1)
         install.finalize_options(self)
 
-version = "1.5.0"
-dependencies = ["cffi>=1.1.0; python_implementation != 'PyPy'"]
+version = "1.5.1"
+dependencies = ["cffi>=1.1.0"]
```

Comment:
Is there a reason here to drop the `!= 'PyPy'` constraint?

Reply:
ah, see above - needed it to work on pypy and it now does.

Reply:
Oh, is this actually part of the cffi spec? My bad.